src/odemis/driver/actuator.py

```diff
@@ -2188,6 +2188,7 @@ def terminate(self):
 
 # Possible movements of the stage Z axis
 MOVE_DOWN, NO_ZMOVE, MOVE_UP = -1, 0, 1
+ATOL_LENS_POS = 100e-6
```

Comment:
Please indicate the unit (m) in a comment. Also, you should have a separate value for the rotation, for checking rx/rz. Maybe ATOL_LINEAR_LENS_POS and ATOL_ROTATION_LENS_POS. The second on in rad. 1mrad (1e-3) should probably be already quite small (<0.1°)